webpack.mix.js

```diff
@@ -58,5 +63,12 @@ if (!mix.inProduction()) {
 
 // Hash and version files in production.
 if (mix.inProduction()) {
-  mix.version();
+  mix.version()
+     .purgeCss({
```

Comment:
IIRC `laravel-mix-purgecss` is set up so that by default it only runs in production, so I don't think this *needs* to be inside the conditional. Probably no harm in it, though, and is arguably easier for users to understand.

Reply:
that's correct. haven't tested it yet, but I think that goes for [`.version()` as well](https://github.com/JeffreyWay/laravel-mix/blob/master/src/components/Version.js#L35-L37)

Reply:
I haven't been able to find anything that says that for `version()`. In fact [this doc page](https://laravel.com/docs/5.8/mix#versioning-and-cache-busting) seems to indicate the opposite.

Reply:
Just tested this for sanities sake– it does appear `.version()` isn't used outside of production which is in-line with the link I included to the code above. I suspect the doc page is similar to @mmirus's argument above of making it more-so easier to understand for the user.

Reply:
Didn't know that about `.version()`!

Reply:
Interesting. Good to know.

Reply:
I just removed these since though it does make it clear it’s in production, we can also set `sourceMaps(false)` to do the same thing and reduce a bunch of boilerplate.